core/src/BibleEngine.class.ts

```diff
@@ -998,7 +1022,7 @@ export class BibleEngine {
                 //        since an indent is a block group and a linebreak at the end of a block,
                 //        shouldn't have an effect. If this causes a problem, we will need to
                 //        implement some forward or backward looking magic, which is complex.
-                if (content.groupType === 'indent' || content.groupType === 'poetry')
+                if (content.groupType === 'indent' || content.groupType === 'linegroup')
                     globalState.phraseStack[globalState.phraseStack.length - 1].linebreak = true;
```

Comment:
Is it safe to assume that phrases won't be empty here?

Reply:
added a check